src/main/java/frc/robot/vision/filters/VisionFilters.java

```diff
@@ -0,0 +1,51 @@
+package frc.robot.vision.filters;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.robot.constants.Field;
+import frc.robot.vision.data.AprilTagVisionData;
+import frc.robot.vision.data.VisionData;
+import frc.utils.Filter;
+
+public class VisionFilters {
+
```

Comment:
repetitive code. consider adding `isRotationInTolerance` to delete half of the characters here.

Reply:
there are only 2 funcs that use rotations, no need